exosphere-shared/src/docker-helper.ls

```diff
@@ -11,7 +11,10 @@ class DockerHelper
 
 
   @container-is-running = (container-name) ->
-    child_process.exec-sync('docker ps --format {{.Names}}/{{.Status}}') |> (.to-string!) |> (.split '\n') |> (.includes "#{container-name}/Up")
+    running-containers = child_process.exec-sync('docker ps --format {{.Names}}/{{.Status}}') |> (.to-string!) |> (.split '\n') 
```

Comment:
Let's split by https://nodejs.org/api/os.html#os_os_eol instead of hard-conding the string. It is different on Windows.